src/main/java/org/tron/common/overlay/discover/node/NodeManager.java

```diff
@@ -61,7 +66,7 @@
   static final int MAX_NODES = 2000;
   static final int NODES_TRIM_THRESHOLD = 3000;
 
-  Consumer<DiscoveryEvent> messageSender;
+  Consumer<UdpEvent> messageSender;
 
   NodeTable table;
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Explicitly declare the visibility for "table". ([squid:S2039](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS2039))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=951&issues=AWPTJmwzA0k93xG_IQRf&open=AWPTJmwzA0k93xG_IQRf)

[ISSUE_KEY]: AWPTJmwzA0k93xG_IQRf